client/src/Pages/Profile/SideBar/index.js

```diff
@@ -20,17 +21,22 @@ function a11yProps(index) {
 function SideBar() {
   const classes = useStyles();
   const [value, setValue] = useState(0);
+  const [username, setUsename] = useState('');
 
   const handleChange = (event, newValue) => {
     setValue(newValue);
   };
 
+  function getUserName(_username) {
```

Comment:
You probably don't need to define this function, you could pass setUsername directly instead